src/cascadia/TerminalApp/MinMaxCloseControl.xaml

```diff
@@ -10,12 +10,283 @@
+            <SolidColorBrush x:Key="CloseStrokeBrushPointerOver" Color="White"/>
+            <SolidColorBrush x:Key="CloseBackgroundBrushPressed" Color="#f1707a"/> <!-- Derived by color-picking a clicked close button. -->
+            <SolidColorBrush x:Key="CloseStrokeBrushPressed" Color="Black"/>
+
+            <!-- We need a custom template for every caption button because templates cannot be inherited,
+                 and we need to change what happens to specific buttons when they transition between visual
+                 states. As of 19H1, the XAML Button defaults to a tilt animation when it's clicked.
+                 We're overriding its VisualState(s) to suppress that here.
+
+                 Other Style properties (which *will* be inherited) include whether the button is a tab stop
+                 and its background color. The caption buttons cannot be tab stops, because no other win32
+                 application works like that.
+            -->
+            <Style x:Key="CaptionButton" TargetType="Button">
+                <Setter Property="Background" Value="{ThemeResource ApplicationPageBackgroundThemeBrush}" />
+                <Setter Property="IsTabStop" Value="False" />
+                <Setter Property="Template">
+                    <Setter.Value>
+                        <ControlTemplate TargetType="Button">
+                            <ContentPresenter x:Name="ButtonBaseElement"
+                                Background="{TemplateBinding Background}"
+                                BackgroundSizing="{TemplateBinding BackgroundSizing}"
+                                BorderBrush="{TemplateBinding BorderBrush}"
+                                BorderThickness="{TemplateBinding BorderThickness}"
+                                Content="{TemplateBinding Content}"
+                                ContentTemplate="{TemplateBinding ContentTemplate}"
+                                ContentTransitions="{TemplateBinding ContentTransitions}"
+                                CornerRadius="{TemplateBinding CornerRadius}"
+                                Padding="{TemplateBinding Padding}"
+                                HorizontalContentAlignment="{TemplateBinding HorizontalContentAlignment}"
+                                VerticalContentAlignment="{TemplateBinding VerticalContentAlignment}"
+                                AutomationProperties.AccessibilityView="Raw">
+
+                                <VisualStateManager.VisualStateGroups>
+                                    <VisualStateGroup x:Name="CommonStates">
+                                        <VisualState x:Name="Normal" />
+
+                                        <VisualState x:Name="PointerOver">
+                                            <Storyboard>
+                                                <ObjectAnimationUsingKeyFrames Storyboard.TargetName="ButtonBaseElement" Storyboard.TargetProperty="Background">

```

Comment:
Recommend using VisualState.Setters syntax -- we only don't do that in generic.xaml for historical reasons but we're trying to switch over to the more compact syntax.